agent/src/bpf/audit.bpf.c

```diff
@@ -255,4 +383,21 @@ BPF_USDT(blob_data, long context, const char *key_ptr)
   return record_blob_data(ctx, context, key_ptr);
 }
 
+SEC("usdt")
+int
+BPF_USDT(data, long context, const struct crypto_auditing_data *array_ptr,
```

Comment:
the name "data" sounds overly generic, is this intentional? would something more specific like "composite" fit better?

Reply:
It's on purpose as we recommend using this interface rather than `string_data`, `word_data`, and `blob_data`.

Reply:
OK